### PR TITLE
Fix copy/paste typos

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix copy/paste typos.

fixes https://github.com/eclipse/che/issues/21744

### CODE REVIEW

File: packages/dashboard-frontend/src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx [hunk 1]
```diff
@@ -246,7 +246,7 @@ export class StorageTypeFormGroup extends React.PureComponent<Props, State> {
           name="per-workspace"
           id="per-workspace-type-radio"
           description="Per-workspace Storage. One PVC is provisioned for each workspace within the namespace.
-        All of the workspace's storage are mounted on subpaths within the workspace's PVC."
+          All of the workspace's storage are mounted on subpaths within the workspace's PVC."
```
Comment (ibuziuk) [inline packages/dashboard-frontend/src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx:249]:
```suggestion
```

**Review by akurinnoy — APPROVED** [review]

**Comment by ibuziuk** [comment]
@olexii4 this PR is related to https://github.com/eclipse/che/issues/21744, right?

**Comment by olexii4** [comment]
> @olexii4 this PR is related to [eclipse/che#21744](https://github.com/eclipse/che/issues/21744), right?

Yes, you are right.

File: packages/dashboard-frontend/src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx [hunk 2]
```diff
@@ -230,8 +230,8 @@ export class StorageTypeFormGroup extends React.PureComponent<Props, State> {
           label="Per-user"
           name="per-user"
           id="per-user-type-radio"
-          description="Per-workspace Storage. A PVC is provisioned for each workspace within the namespace.
-        All of the workspace's storage are mounted on subpaths within the workspace's PVC."
+          description="Per-user Storage. One PVC is provisioned per namespace.
+          All of the workspace's storage (volume mounts) mounted in it on subpaths according to devworkspace ID."
```
Comment (ibuziuk) [inline packages/dashboard-frontend/src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx:234]:
```suggestion
```
I would probably remove this line two


Another option would be changing it to

**Review by ibuziuk — APPROVED** [review]

**Comment by ibuziuk** [comment]
@olexii4 thank you, once merged could you please backport to 7.54.x 
after that we can close https://github.com/eclipse/che/issues/21744

**Pull request merged**